### ISSUE TITLE
parameters for training

### ISSUE THREAD

**Issue by liupengcnu** [Tencent/HunyuanDiT#82]
Hi, can you share the value of some training parameters, like the timesteps sample arrange, the learning rate in training?  Thankyou~

**Comment by zml-ai** [comment]
Hi, the recommended parameters for our full-parameter training, LoRa training, and ControlNet training are all included in their respective training scripts. Feel free to consult them.

Full-parameter: https://github.com/Tencent/HunyuanDiT/blob/main/hydit/train.sh
Controlnet: https://github.com/Tencent/HunyuanDiT/blob/main/hydit/train_controlnet.sh 
Lora：https://github.com/Tencent/HunyuanDiT/blob/main/lora/train_lora.sh
